subworkflows/local/utils_nfcore_nallo_pipeline/main.nf

```diff
@@ -653,3 +653,8 @@ def createReferenceChannelFromSamplesheet(param, schema, defaultValue = '') {
     return param ? Channel.fromList(samplesheetToList(param, schema)) : defaultValue
 }
 
+def validatePacBioLicense() {
+    if (params.phaser.matches('hiphase') && params.preset == 'ONT_R10') {
```

Comment:
This License also applies to HiFiCNV. I couldn't find a check that ensures that it is not run with ONT data. Considering that it is the only CNV caller we currently support, we should look for an alternative.

Reply:
I just found the issue pointing out this concern (#447) so I'll mark this as resolved

Reply:
Yeah, let's stay on 1.0.0 for a bit 😄